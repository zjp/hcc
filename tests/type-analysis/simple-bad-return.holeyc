int glob;
void main(){
	int a;
	glob = a;
	return a;
}
