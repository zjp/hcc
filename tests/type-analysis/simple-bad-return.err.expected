FATAL [5,9]: Return with a value in void function
Type Analysis Failed
